omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -37,6 +37,9 @@
     function handlePatientRowSelection() {
     	var afterSelectedUrl = '${ ui.escapeJs(config.afterSelectedUrl) }';
         this.handle = function (row) {
+            if(!row || !row.uuid) {
```

Comment:
Logically, this should just be:
`if(!row)`
Because you use '||'. And if there is `row.uuid`, then, there is `row`